schedule/migration/migration_autoyast.yaml

```diff
@@ -2,6 +2,7 @@
 schedule:
   - autoyast/prepare_profile
   - migration/version_switch_origin_system
+  - installation/bootloader
   - boot/boot_to_desktop
   - console/scc_cleanup_reregister
```

Comment:
we don't need this one if we un-register in the parent. AutoYaST already register the system for us, we don't need to do it explicitly.